### PR TITLE
[fix][doc] Fix sql-hive and hive-cli doc 

### LINKED ISSUE #12764: [Doc][hive-sql] hive-sql and hive-cli doc not approperiate
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

problems:
hive in the sql task type, cannot execute multi-statements and use ';'.
hive-cli doc can execute multi-statements, need add execute multi-statements case snapshoot.

### Documentation Links

https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/task/sql.html
https://dolphinscheduler.apache.org/zh-cn/docs/latest/user_doc/guide/task/hive-cli.html

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Tianqi-Dotes**
## Purpose of the pull request

close #12764

## Brief change log
fix doc:
hive in the sql task type, cannot execute multi-statements and use ';'.
hive-cli doc can execute multi-statements, need add execute multi-statements case snapshoot.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/en/guide/task/sql.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Refer to [datasource-setting](../howto/datasource-setting.md) `DataSource Center
 #### Create a Temporary Table in Hive and Write Data
 
 This example creates a temporary table `tmp_hello_world` in Hive and writes a row of data. Before creating a temporary table, we need to ensure that the table does not exist. So we use custom parameters to obtain the time of the day as the suffix of the table name every time we run, this task can run every different day. The format of the created table name is: `tmp_hello_world_{yyyyMMdd}`.
+**Note**：the hive datasource in the SQL task based on JDBC to execute SQLs, SQL statement does not support multi-statements, please avoid using ';' at the end of the statement. To process multi-statements please use Hive-Cli task.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/sql.md]: should we also remove this statement?
image

Comment (zhongjiajie) [reply docs/docs/en/guide/task/sql.md]: And BTW, can we use hyperlink here? 

```
To process multi-statements please use [Hive-Cli](./hive-cli.md) task.
```

Comment (Tianqi-Dotes) [reply docs/docs/en/guide/task/sql.md]: done

**Review by zhongjiajie — DISMISSED** [review]
LGTM, thanks

**Review by EricGao888 — DISMISSED** [review]

**Review by zhongjiajie — APPROVED** [review]

**Review by jieguangzhou — APPROVED** [review]
LGTM

**Pull request merged**